### PR TITLE
Feat/met 6060 pass debuplication reason to ecloud and restore back during tombstoning

### PR DESCRIPTION
**Pull request by stzanakis**

### CODE REVIEW

File: metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/DepublishPlugin.java [hunk 1]
```diff
@@ -63,6 +63,8 @@ public DpsTask prepareDpsTask(String datasetId,
         extraParameters.put(PluginParameterKeys.RECORD_IDS_TO_DEPUBLISH, recordIdList);
       }
     }
+    //TODO: 2024-09-24 - Update below key with the PluginParameterKeys equivalent when it's available
```
Comment (jeortizquan) [inline metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/DepublishPlugin.java:66]: is there a ticket for this part?

**Review by jeortizquan — COMMENTED** [review]
two observations.

File: metis-indexing/src/main/java/eu/europeana/indexing/IndexedRecordAccess.java [hunk 2]
```diff
@@ -60,7 +63,7 @@ public long countRecords(String datasetId) {
   }
 
   /**
-   * Get fullbean from database give the rdf about.
+   * Get fullbean from database given an rdf about.
```
[the remark this answers is not in the document]
Comment (stzanakis) [reply metis-indexing/src/main/java/eu/europeana/indexing/IndexedRecordAccess.java:66]: That is not valid grammar, "an" is correct in this case.

File: metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/DepublishPlugin.java (hunk 1 above)
Comment (stzanakis) [reply metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/DepublishPlugin.java:66]: Ok I just created 6182 part of the depublication topology preparations(6147) and also 6183 to cleanup this and the deprecations mentioned earlier this morning

File: metis-indexing/src/main/java/eu/europeana/indexing/IndexedRecordAccess.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeortizquan) [reply metis-indexing/src/main/java/eu/europeana/indexing/IndexedRecordAccess.java:66]: :thinking: i just knew one rule about `a` and `an` seems there are more rules. I learned new stuff.
```
 Articles with Acronyms, a or an

Finally, the rule applies to acronyms as well. If you pronounce a letter as a letter and it begins with a vowel sound, you should precede it with an. The consonants with vowel sounds include f, h, l, m, n, r, s, and x.

    He flew in an SST. He fired an M‑1. He attended an FDA hearing.
```

**Review by jeortizquan — APPROVED** [review]

File: metis-indexing/src/main/java/eu/europeana/indexing/IndexedRecordAccess.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (stzanakis) [reply metis-indexing/src/main/java/eu/europeana/indexing/IndexedRecordAccess.java:66]: Nice!

**Pull request merged**